### ISSUE TITLE
Level patches don't apply after switching levels.

### ISSUE THREAD

**Issue by DreamyCecil** [DreamyCecil/AdvancedMultiplayer2#17]
After going to a level that's supposed to apply custom patches (from LevelPatches folder), the patches don't work. The issue is possibly due to the patch being applied upon starting the game (perhaps in `CGame::NewGame()`), i.e. after loading the world for the first time.

But then after going to a level that's not supposed to have patches from the level that does, it still applies them (the class patches but not the dummy patches).

Basically the class patching remains consistent no matter what but should instead be updated each time the world is loaded.
